index.d.ts

```diff
@@ -305,6 +305,17 @@ export interface Theme {
 }
 
 export const defaultTheme: Theme
+export const classicTheme: Theme
+
+interface DeprecatedDefaultTheme {
+  colors: Colors,
+  scales: ColorScales,
+  typography: Typography,
```

Comment:
FYI I had removed stuff from `Typography` (`headings` I think).

Reply:
looks like headings and text! will add back.